src/js/pages/catalog/PackagesTab.js

```diff
@@ -208,16 +213,27 @@ class PackagesTab extends mixin(StoreMixin) {
       </p>
     );
     let title = "Community";
-    const isSearchActive = this.state.searchString !== "";
     const titleClasses = classNames("flush-top", {
       short: !isSearchActive,
       tall: isSearchActive
     });
 
     if (isSearchActive) {
       const foundPackagesLength = packages.getItems().length;
-      const packagesWord = StringUtil.pluralize("service", foundPackagesLength);
+      if (foundPackagesLength < 1) {
+        const noResults = `No results were found for your search: "${
+          this.state.searchString
+        }" `;
+
+        return (
+          <div className="clearfix">
+            {noResults}
+            (<a onClick={this.clearInput}>view all</a>)
```

Comment:
This should be `cursor: pointer`

Reply:
You may use `className="clickable"` for convenience